packages/block-library/src/button/edit.native.js

```diff
@@ -372,7 +372,6 @@ class ButtonEdit extends Component {
 						__unstableMobileNoFocusOnMount={ ! isSelected }
 						selectionColor={ textColor }
 						onBlur={ () => {
-							this.onToggleButtonFocus( false );
```

Comment:
Could you explain to me why this change is needed?

Reply:
I found this line of code redundant because it's introducing unnecessary state change - button is still selected and after closing the bottom sheet `RichText` is focused so there is no need to change that local state value. Additionally, styling buttons on the toolbar above the keyboard was disappearing:

before | after
--- | ---
<img src="https://user-images.githubusercontent.com/22746080/89562464-46fdaa00-d81a-11ea-9023-0ceac4eb37df.gif" width="300" /> | ![ok](https://user-images.githubusercontent.com/22746080/89562480-4cf38b00-d81a-11ea-9d88-222c1bb16e13.gif)

Reply:
Thanks for the details!